### PR TITLE
Add Apache Camel icon for the Stack (#11236)

### PR DESCRIPTION
**Pull request by apupier**
Signed-off-by: Aurélien Pupier <apupier@redhat.com>

### CODE REVIEW

**Comment by apupier** [comment]
ci-test

File: ide/che-core-ide-stacks/src/main/resources/stacks-images/type-apache-camel.svg [hunk 1]
```diff
@@ -0,0 +1,4297 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (benoitf) [inline ide/che-core-ide-stacks/src/main/resources/stacks-images/type-apache-camel.svg:1]: missing license/origin in the SVG

Comment (benoitf) [reply ide/che-core-ide-stacks/src/main/resources/stacks-images/type-apache-camel.svg:1]: also it's strange it seems there is PNG in that SVG

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/835/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/835//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11296
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

File: ide/che-core-ide-stacks/src/main/resources/stacks-images/type-apache-camel.svg (hunk 1 above)
Comment (apupier) [reply ide/che-core-ide-stacks/src/main/resources/stacks-images/type-apache-camel.svg:1]: argh on Apache Camel irc chat, someone pointed me to this "false" svg, just an embedded png :'(
I will ping them again

**Comment by apupier** [comment]
ci-test

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/837/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/837//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11296
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

File: ide/che-core-ide-stacks/src/main/resources/stacks-images/type-apache-camel.svg (hunk 1 above)
Comment (apupier) [reply ide/che-core-ide-stacks/src/main/resources/stacks-images/type-apache-camel.svg:1]: I modified for the png, it is now a real svg.

I added the license but i still have build complaining about missing header:
[INFO] --- license-maven-plugin:3.0:check (check-headers) @ che-core-ide-stacks ---
[INFO] Checking licenses...
[WARNING] Unknown file extension: /home/codenvy/workspace/che-pullrequests-test-ocp/ide/che-core-ide-stacks/src/main/resources/stacks.json
[WARNING] Missing header in: /home/codenvy/workspace/che-pullrequests-test-ocp/ide/che-core-ide-stacks/src/main/resources/stacks-images/type-apache-camel.svg

Comment (benoitf) [reply ide/che-core-ide-stacks/src/main/resources/stacks-images/type-apache-camel.svg:1]: yes plugin is checking for Red Hat / EPL license

you need to exclude your file from the plug-in
https://github.com/eclipse/che/blob/ad6376a43644179ea04b0156b732e468855d5d9c/ide/che-core-ide-stacks/pom.xml#L33-L35

**Comment by apupier** [comment]
ci-test

**Comment by benoitf** [comment]
ci-build

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/838/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/838//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11296
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Comment by apupier** [comment]
seems to work fine :-)

image

File: ide/che-core-ide-stacks/src/main/resources/stacks-images/type-apache-camel.svg (hunk 1 above)
Comment (apupier) [reply ide/che-core-ide-stacks/src/main/resources/stacks-images/type-apache-camel.svg:1]: fixed, thanks for the help

**Review by benoitf — APPROVED** [review]

**Pull request merged**